libs/elements/layout/texts/src/lib/elewa-text-content-item/elewa-text-content-item.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from '@angular/core';
+import { ContentText } from '@elewa-website/models/schema/ui/texts';
+import { Content as mockContent } from '@elewa-website/data/ui/content-text';
+
+
+@Component({
+  selector: 'elewa-website-elewa-text-content-item',
+  templateUrl: './elewa-text-content-item.component.html',
+  styleUrls: ['./elewa-text-content-item.component.scss'],
+})
+export class ElewaTextContentItemComponent {
+  @Input() textContent!: ContentText []; 
+  Content: ContentText[];
+
+  constructor () {
+    this.Content = mockContent;
+  }
```

Comment:
We only need the textContent property over here. which should be of type `contentText`, it's not a list.